Top10Words/DataFromBook.cs

```diff
@@ -0,0 +1,30 @@
+namespace Top10Words;
+
+public class DataFromBook
+{
+    private string _fileName;
+    private IEnumerable<string> _bookText;
+    public DataFromBook(string fileName)
+    {
+        _fileName = fileName;
+        _bookText = File.ReadLines(_fileName);
+    }
+    
+    public int GetAllWordsCount()
```

Comment:
Please create a separate class which will be responsible for a book processing and word calculation.
We have to do that because we are going to implement other approaches for calculation and then compare performance.